frontend/src/scenes/insights/filters/ActionFilter/ActionFilterRow/ActionFilterRow.tsx

```diff
@@ -348,13 +348,33 @@ export function ActionFilterRow({
                                                 renderValue={(currentValue) => (
                                                     <Tooltip
                                                         title={
-                                                            <>
-                                                                Calculate{' '}
-                                                                {mathDefinitions[math ?? ''].name.toLowerCase()} from
-                                                                property <code>{currentValue}</code>. Note that only{' '}
-                                                                {name} occurences where <code>{currentValue}</code> is
-                                                                set with a numeric value will be taken into account.
-                                                            </>
+                                                            currentValue === '$session_duration' ? (
+                                                                <>
+                                                                    Calculate{' '}
+                                                                    {mathDefinitions[math ?? ''].name.toLowerCase()} of
+                                                                    the Session Duration. Unlike other properties this
+                                                                    is based on the
+                                                                    <code>$session_id</code> property associated with
+                                                                    events. The duration is derived from the time
+                                                                    difference between the first and last event for each
+                                                                    distinct
+                                                                    <code>$session_id</code>.
+                                                                    <br />
+                                                                    Learn more about how PostHog tracks sessions in{' '}
+                                                                    <a href="https://posthog.com/docs/user-guides/sessions">
```

Comment:
We can't put links in tooltips, because it's not possible to hover _into_ the tooltip.

Reply:
Oh yeah good point